support-workers/src/main/scala/com/gu/emailservices/SupporterPlusEmailFields.scala

```diff
@@ -20,6 +23,14 @@ class SupporterPlusEmailFields(
       state.accountNumber,
       created,
     ).map { paymentFields =>
+      import state._
+      val promotion = paperFieldsGenerator.getAppliedPromotion(
+        state.promoCode,
+        state.user.billingAddress.country,
+        ProductTypeRatePlans.supporterPlusRatePlan(state.product, touchPointEnvironment).map(_.id).getOrElse(""),
+      )
+      val subscription_details = SubscriptionEmailFieldHelpers
+        .describe(paymentSchedule, product.billingPeriod, product.currency, promotion)
```

Comment:
this line generates the nice "£blah per month for 3 months then £hello per month" human readable from the schedule that zuora generates